### PR TITLE
Document custom sbt commands

### LINKED ISSUE #191: Document custom sbt commands in README
Derived from https://github.com/rexim/Morganey/pull/185#issuecomment-246267603

### PR DESCRIPTION
**Pull request by keddelzz**
closes  #191

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,25 @@ $ sbt assembly
...
+### Custom sbt commands ###
+
+This project introduces the following custom sbt commands:
+
+- `sbt build`: Compiles the subprojects `kernel`, `macros` and
+  `stdlib` and copies the resulting jar-files into the target directory
+  of the root project.
+
+- `sbt buildModuleIndex`: Traverses the stdlib subproject and creates
+  a resource file containing a list of modules - the so called
```
Comment (ForNeVeR) [inline README.md]: `so called` → `so-called`

File: README.md [hunk 2]
```diff
@@ -111,6 +111,25 @@ $ sbt assembly
 
 After that find the uberjar at `target/scala-2.11/morganey.jar`
 
+### Custom sbt commands ###
+
+This project introduces the following custom sbt commands:
+
+- `sbt build`: Compiles the subprojects `kernel`, `macros` and
+  `stdlib` and copies the resulting jar-files into the target directory
```
Comment (ForNeVeR) [inline README.md]: May I suggest `jar-files` → `JAR files`?

File: README.md [hunk 3]
```diff
@@ -111,6 +111,25 @@ $ sbt assembly
...
+  of the root project.
+
+- `sbt buildModuleIndex`: Traverses the stdlib subproject and creates
+  a resource file containing a list of modules - the so called
+  [module-index]. This command is automatically run if you `sbt
```
Comment (ForNeVeR) [inline README.md]: I don't like "is automatically run", is it right? I'd say "is automatically executed". I like to execute ,3

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -111,6 +111,25 @@ $ sbt assembly
...
+  compile` the project.
+
+- `sbt rebuild`: Alias for the execution of the commands `sbt clean`
+  and `sbt build`.
+
+- `sbt retest`: Alias for the execution of the commands `sbt rebuild`
+  and `sbt retest`.
```
Comment (rexim) [inline README.md]: `sbt test`. The current documentation makes the command seem recursive. :)

**Review by rexim — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (rexim) [reply README.md]: Do I understand correctly that you admit that this comment is your subjective opinion and the phrase doesn't have any critical problems with meaning or grammatics?

Comment (keddelzz) [reply README.md]: I think there's also a word-order error in my sentence. It should read "This command is run/executed automatically".

**Review by rexim — APPROVED** [review]

**Review by ForNeVeR — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (ForNeVeR) [reply README.md]: I'm still confused by using "run" in a passive voice, but I've checked in grammar and that's correct. So, the problem is fixed.

Comment (rexim) [reply README.md]: > I'm still confused by using "run" in a passive voice

Damn irregular verbs!

**Comment by rexim** [comment]
I think we have to finally fix our unstable CI. I captured the scope of that in #299 and will try to do an investigation in the nearest future.

CI failures should not affect the mergability of this PR. Will restart the builds until they pass.

**Comment by rexim** [comment]
@keddelzz thank you so much for the changes!
@ForNeVeR thank you for the review!

**Pull request merged**